### DISCUSSION TITLE
[Design/Antagonist] Syndicate Informant

### DISCUSSION THREAD

**Discussion by datlo** [ParadiseSS13/Paradise#21868] [Documents]
Collaborators' execution was flawed because it leads to self-antagging way too often. Beneath that, I think the core idea of it isn't necessarily bad. This is my attempt at bringing it back, by making it a full fledged antagonist, with antag rights.

Feedback is appreciated. I'll hopefully code this in the future when I find the time/motivation, but if anyone wants to take their own stab at it, *please do*, it's really hard for me to find moments to code spessmen these days.

-------------

Syndicate Informant

Only one per round, replaces one traitor in gamemodes with traitors, with priority to experienced players using a weighted roll. Functions as an IC antag mentor. Doesn't carry an uplink. EDIT : Some people dont like the idea of weighted roll. Can just be a playtime requirement instead.

The informant is not aware of any traitors onboard, but starts with a syndicate radio key to make contact, they also know syndicate code words.

Traitor players do not know the informant's identity, but will be told that there is one they can contact by purchasing a syndicate radio key. Less experienced antag players will be told the Informant's identity directly, so that they may go to them for help.

Objectives : 
1. Escape aboard the escape shuttle.  Do not use escape pods. (Lore reason in intro blurb : You're needed to monitor outgoing traffic for the syndicate)
2. Assist syndicate agents, and any other entity hostile to NT and willing to cooperate.
3. Optional : If possible, stay out of NT's suspicions so that we may keep you undercover for future operations.

Objective 2 and 3 automatically greentext if you escape successfully, they're there to guide your gameplay.

The informant, despite its lack of inherent objectives other than escape, is a full antagonist, allowed to perform any actions allowed to antagonists, unlike the old collaborator system.

This means that informant players can be as loud as they wish as long as they're assisting other antags with their own objectives. 
However, they're incentivized to stay undercover by the requirement to escape using the shuttle and not a pod.

If no one buys syndicate radio keys, the Informant can still be on the lookout for people that are suspicious, or suspected by security, and make contact that way with code words. They can also breakout perma prisonners.

Jobs that can roll Informant : Quartermaster, Doctor, Paramedic, Coroner, Chemist, Engineer, Atmos Tech, Scientist, Clown.

These changes together should result in an increase in antag cooperation, better experience for newer antag players, and a slight decrease in antag chaos, as rounds will have one less uplink and set of objectives since the Informant replaces a Traitor.

**Comment by McRamon** [comment]
Why job limitation

**Reply by datlo** [reply]
The idea of the antag is to have the player use the tools of the department he's in to assist antags while staying undercover. With the exception of clown because no one's gonna be surprised to see a clown breaking into everywhere. Cargo tech could be added to the list aswell, I just kinda forgor em.
That restriction can be removed if preferred.

**Comment by DGamerL** [comment]
I get that the jobs are probably limited because of how useful someone can be, but other jobs also get good benefits, or to show you can still be a good and fun Antag without extra access and tools

For the rest, I never played during collaborators so I'm interested to see how this will go!

**Comment by GDNgit** [comment]
How do you see informants playing their round out? The things that jump out at me right away are
1. What is preventing informants from just being a "break out of perma" role/a role that just steals from evidence and starts antagging?
2. Are they just supposed to be like a designated tots that goals are pretty much to revive/arm antagonists?
3. Can they be rolled for kill objectives? If they can't that would make them 100% trustworthy which is wack
4. Why shouldn't they just go along and antag with a tot, encouraging people to get into tot congas and try and steamroll sec
